### PR TITLE
[4.12-4.14] GCP marketplace image for both compute and control-plane machines

### PR DESCRIPTION
**Pull request by jianli-wei**
- use GCP marketplace RHCOS image for both compute machines and control-plane machines

### CODE REVIEW

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-gcp

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-gcp

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.12-installer-rehearse-gcp

**Comment by jianlinliu** [comment]
cc @liangxia to review

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -2584,6 +2584,31 @@ tests:
...
     - chain: openshift-e2e-test-qe-longduration
     workflow: cucushift-installer-rehearse-gcp-ipi-filestore-csi
+- as: gcp-ipi-marketplace-f28
+  cron: 47 15 17 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_OSIMAGE: projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-413-x86-64-202305021736
+      CONTROL_PLANE_OSIMAGE: projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-413-x86-64-202305021736
+      E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @network-ovnkubernetes'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:2594]: Shall we add `and not @fips`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 2]
```diff
@@ -2584,6 +2584,31 @@ tests:
...
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-gcp-ipi-workers-marketplace
+- as: gcp-ipi-marketplace-f28-destructive
+  cron: 31 21 18 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_OSIMAGE: projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-413-x86-64-202305021736
+      CONTROL_PLANE_OSIMAGE: projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-413-x86-64-202305021736
+      E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @network-ovnkubernetes'
```
Comment (gpei) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:2606]: And also here for `and not @fips`

**Pull request merged**